CoreSchemaView/src/au/gov/asd/tac/constellation/views/schemaview/providers/VertexTypeNodeProvider.java

```diff
@@ -223,79 +223,41 @@ private void populateTree() {
         treeView.setRoot(root);
     }
 
-    private boolean isFilterMatchTextCurrentNodeOrAnyChildren(SchemaVertexType treeItem) {
-        boolean found = isFilterMatchText(treeItem.getName());
-        if (!found) {
-            found = isFilterMatchTextAnyProperty(treeItem);
-        }
-        if (!found) {
-            found = isFilterMatchTextAnyChildNodes(treeItem);
-        }
-        return found;
+    private boolean isFilterMatchCurrentNodeOrAnyChildren(SchemaVertexType treeItem) {
+        return filterText.getText().toLowerCase().isEmpty() ? false
+                : isFilterMatchText(treeItem.getName()) || isFilterMatchAnyProperty(treeItem) || isFilterMatchAnyChildNodes(treeItem);
```

Comment:
Change this return to `!filterText.getText().isEmpty() || isFilterMatchText(treeItem.getName()) || isFilterMatchAnyProperty(treeItem) || isFilterMatchAnyChildNodes(treeItem)`

Reply:
@antares1470  I added original line to avoid further processing when the strings are null. Feels like the above will still try to process when it is null?

Reply:
see above comment

Reply:
Similarly,  && seems to work here.